plugins/optimization-detective/storage/class-od-rest-url-metrics-store-endpoint.php

```diff
@@ -0,0 +1,347 @@
+<?php
+/**
+ * REST API integration for the plugin: OD_REST_URL_Metrics_Store_Endpoint.
+ *
+ * @package optimization-detective
+ * @since 0.1.0
+ */
+
+// @codeCoverageIgnoreStart
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+// @codeCoverageIgnoreEnd
+
+/**
+ * OD_REST_URL_Metrics_Store_Endpoint class
+ *
+ * @since n.e.x.t
+ * @access private
+ */
+final class OD_REST_URL_Metrics_Store_Endpoint {
+
+	/**
+	 * Namespace for the REST API endpoint.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	const REST_API_NAMESPACE = 'optimization-detective/v1';
```

Comment:
Should this just be:

```suggestion
	const NAMESPACE = 'optimization-detective/v1';
```

Reply:
Update with - https://github.com/WordPress/performance/pull/1865#discussion_r1991808494